subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/RepositoryHandler.java

```diff
@@ -88,7 +88,17 @@
      * @since 4.4
      */
     @Incubating
-    ArtifactRepository gradlePluginPortal();
+    MavenArtifactRepository gradlePluginPortal();
+    
+    /**
+     * Adds a repository which looks in Gradle Central Plugin Repository for dependencies.
+     * 
+     * @param action a configuration action
+     * @return the added resolver
+     * @since 5.3
+     */
+    @Incubating
+    MavenArtifactRepository gradlePluginPortal(Action<? super MavenArtifactRepository> action);
```

Comment:
For consistency, and because having the support for `mavenContent` really does not mean a lot in this case, I would use `ArtifactRepository` here.